Documentation~/Toggle_X_Ray.md

```diff
@@ -0,0 +1,10 @@
+# Toggle X Ray 
+
+Shows hidden faces.
+
+<!--
+Note to reviewers: this is a placeholder; the page itself will be written in a different PR. 
+
+[[From the what's new page: * Added a new [Selection X-Ray](preferences.md#sel-xray) option to highlight hidden element selections with a muted color. The default shortcut is **Ctrl/Alt+Shift+X** (modifiable in the Shortcuts Manager), and you can also access it through the ProBuilder menu (**Tools** > **ProBuilder** > **Interaction** > **Toggle X Ray**).]]
+
+[[why is this both a preference an an action?]]-->
```

Comment:
I don't really know much about this feature. @modrimkus, can you explain this one?

Reply:
@suearkinunity It toggles if selected elements that are occluded by geometry are still visualized.

https://github.com/user-attachments/assets/927cd67f-e3a8-42a5-832b-cd6fb5b22d52

For why it's a pref but also has a shortcut - I'm not sure. @gabriel-unity thoughts? : )

Reply:
@gabrielw-us thoughts? : )